src/Private/Security.ps1

```diff
@@ -180,7 +180,7 @@ function Test-PodeRouteLimit {
     }
 }
 
-function Test-PodeEndpointLimit {
+function Test-PodeEndpointByProtocolTypeLimit {
```

Comment:
This function looks like it might have gotten cause in a `Test-PodeEndpointByProtocol` find+replace :)